persistence/src/main/java/com/epam/esm/dao/creator/GiftCertificateQueryCreator.java

```diff
@@ -17,15 +18,17 @@
 
     private static final String QUERY_WITHOUT_SEARCH_CRITERIA = "SELECT gift_certificates.*, tags.* FROM gift_certificates LEFT JOIN " +
```

Comment:
As I see you have `GiftCertificateQuery.java` class for Gift certificate constants could we move this queries there?

Reply:
I think not, because this string is not a full and final query, and in that class there are static query strings.